src/main/java/jenkins/plugins/sloccount/steps/SloccountPublisherStep.java

```diff
@@ -0,0 +1,168 @@
+package jenkins.plugins.sloccount.steps;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import hudson.Extension; 
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.plugins.sloccount.SloccountPublisher;
+import org.jenkinsci.plugins.workflow.steps.StepDescriptor ; 
+import org.jenkinsci.plugins.workflow.steps.Step; 
+import org.kohsuke.stapler.DataBoundConstructor; 
+import org.kohsuke.stapler.DataBoundSetter;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.io.IOException;
+import java.util.Set;
+import java.io.Serializable;
+import javax.annotation.Nonnull;
+import org.jenkinsci.plugins.workflow.steps.StepContext;
+
+import org.jenkinsci.plugins.workflow.steps.StepExecution;
+import org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution;
+
+/**
+ *
+ * @author piomis
+ */
+public class SloccountPublisherStep extends Step implements Serializable {
```

Comment:
I followed blog post (for Gantling plugin) which described how to add Pipeline support in plugin. There was also such class added - as far as I recall it was mainly to add "special function" in Groovy.
In this case "publishSloccount()"